modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java

```diff
@@ -81,18 +81,12 @@ public void render(
 		super.render(renderRequest, renderResponse);
 	}
 
-	@Reference(unbind = "-")
-	protected void setItemSelector(ItemSelector itemSelector) {
-		_itemSelector = itemSelector;
-	}
+	@Reference
+	protected ItemSelector itemSelector;
```

Comment:
@JanisZhang This portlet doesn't have sub-class, please confirm whether these referenced fields need to be `protected`.

Reply:
@dantewang got it, thx.